src/lib.rs

```diff
@@ -79,8 +80,19 @@ impl PkvStore {
         Self::new_from_config(&config)
     }
 
+    /// Creates or opens a pkv store
+    ///
+    /// Like [`PkvStore::new`], but requires a direct path.
+    /// The `path` is used to create a backing file
+    /// in a corresponding location on the users device.
+    #[cfg(any(sled_backend, rocksdb_backend))]
+    pub fn new_with_path(path: &Path) -> Self {
```

Comment:
Could you make this take `AsRef<Path>`? Would allow passing in string slices etc.

Reply:
Sure, no problem